### PR TITLE
Exposing commands to multitool

### PR DESCRIPTION
**Pull request by eddynaka**

### CODE REVIEW

File: src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("export-validation-rules", HelpText = "Export rules metadata to a SARIF or SonarQube XML file.")]
```
Comment (lgolding) [inline src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs:9]: >or SonarQube XML file [](start = 82, length = 21)

Do we really do that?

Comment (lgolding) [inline src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs:9]: >rules [](start = 56, length = 5)

"rules" => "validation rules"

File: src/Sarif.Multitool.Library/ExportValidationConfigurationOptions.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("export-validation-configuration", HelpText = "Export rule options to an XML or JSON file that can be edited and used to configure subsequent analysis.")]
```
Comment (lgolding) [inline src/Sarif.Multitool.Library/ExportValidationConfigurationOptions.cs:9]: >rule [](start = 64, length = 4)

"rule" => "validation rule"

File: src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs (hunk 1 above)
Comment (eddynaka) [reply src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs:9]: @michaelcfanning, do you know if this exports a SonarQube XML? I saw in the code that we export XML, but I'm not sure if that is a simple XML or not.

---
In reply to: [489726571](https://github.com/microsoft/sarif-sdk/pull/2073#discussion_r489726571) [](ancestors = 489726571)

**Review by lgolding — APPROVED** [review]
:shipit:

**Review by lgolding — APPROVED** [review]
LGTM

File: src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs (hunk 1 above)
Comment (michaelcfanning) [reply src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs]: Yes it does. Or it did, successfully, several years ago when originally authored. I don't know if the format has kept pace with current SonarQube requirements.

---
In reply to: [489730440](https://github.com/microsoft/sarif-sdk/pull/2073#discussion_r489730440) [](ancestors = 489730440,489726571)

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Pull request merged**